### PR TITLE
Create client class

### PR DESCRIPTION
**Pull request by Matt-Warnock**
The test for this isn't working and no errors are being handled.
ticket raised for test fix, ticket to be raised for error handling.

### CODE REVIEW

**Comment by Matt-Warnock** [comment]
Please leave for me to merge :-)

File: README.md [hunk 1]
```diff
@@ -10,6 +10,19 @@ Insure you have a version <version> of node installed with node package manager.
 
 `npm install`
 
-Then to run the server:
+### Setup API key
+
+This app uses [Google maps places API](https://developers.google.com/maps/documentation/places/web-service/search)
+You will need to setup an account and get an API key in order to run this app.
+
+### Create .env file
```
Comment (s-palmer) [inline README.md:18]: Nice addition to the readme

File: src/routes.js [hunk 2]
```diff
@@ -1,8 +1,13 @@
 const express = require('express');
 const app = express();
+const Client = require('../src/client');
 
-app.get('/bars', (req, res) => {
-  res.json({results: []});
+app.get('/bars', async (req, res) => {
+  const client = new Client;
+
```
Comment (s-palmer) [inline src/routes.js:7]: Will this be the route we need to send data to from the frontend?

**Review by s-palmer — APPROVED** [review]
Looks good, happy for it to be merged.

File: src/routes.js (hunk 2 above)
Comment (Matt-Warnock) [reply src/routes.js:7]: This was just temp to get visability on the actual call

Comment (s-palmer) [reply src/routes.js:7]: ok cool

**Pull request merged**